alphadia/constants/default.yaml

```diff
@@ -95,17 +95,20 @@ search_advanced:
 
 calibration:
 
-  # minimum number of times (epochs) the updated calibration target has to been passed
-  min_epochs: 3
+  # minimum number of times the classifier is trained.
+  min_training_iterations: 3
```

Comment:
I think the naming and comment are not very precise. Can we rename it to something similar to min_steps_optimization_lock. And emphasize that this is for acquiring the optimization lock/subset. I think we also need a concise name for this process and use it consistently.